lib/api.js

```diff
@@ -27,16 +30,22 @@ export async function getPostBySlugAndLanguage(slug, language, fields = []) {
       items[field] = content
     }
     if (field === 'images') {
-      items[field] =
-        dataByLanguagePostAndImage[language][slug] ||
-        dataByLanguagePostAndImage['any-language']['any-article']
+      try {
```

Comment:
Вот в этой конструкции не уверен, но отрабатывает как нужно

Reply:
Выглядит, конечно, немного хаково :)
Давай заменим весь try ... catch на: 
```suggestion
items[field] = {
        ...(dataByLanguagePostAndImageResize?.[language]?.[slug] ?? {}),
        ...(dataByLanguagePostAndImageDontResize?.[language]?.[slug] ?? {}),
}
```